src/mds/Server.cc

```diff
@@ -1759,6 +1759,7 @@ void Server::handle_client_request(MClientRequest *req)
     req->releases.clear();
   }
 
+  mds->sessionmap.hit_session(session);
```

Comment:
the call here is temporary for testing.

Reply:
Still for testing or are we keeping this?

Reply:
If we're keeping this, it should go in `handle_client_reply` because requests may be retried which would artificially inflate the load.

Reply:
> Still for testing or are we keeping this?

OK -- I forgot about this altogether in midst of average uptime discussions...

Reply:
> If we're keeping this, it should go in handle_client_reply because requests may be retried which would artificially inflate the load.

Why should load adjustment be done in `Client::handle_client_reply()` (that's the only place where that particular function name exist)?

To avoid spiking up load on op retries it should probably be `Server::reply_client_request()`...

Reply:
> Why should load adjustment be done in Client::handle_client_reply() (that's the only place where that particular function name exist)?

Sorry, I mean Server::early_reply/Server::early_reply. See what we do for `perf_gather_op_latency`.